azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/transport/mqtt/mqtt_transport.py

```diff
@@ -652,6 +662,27 @@ def _disable_c2d_messages(self, callback=None):
         self._trig_add_action_to_pending_queue(action)
         self.feature_enabled[constant.C2D_MSG] = False
 
+    def _enable_methods(self, callback=None, qos=1):
```

Comment:
>qos=1 [](start = 45, length = 5)

i don't think this should be a parameter. it should be "fixed" in a constant defined somewhere.
Also we should double-check with the service team that they expect qos 1 and not 0 here (ask Max Gortman)

Reply:
I could get behind this, but only if it were transport wide.

---
In reply to: [267185061](https://github.com/Azure/azure-iot-sdk-python-preview/pull/61#discussion_r267185061) [](ancestors = 267185061)